docs/pages/serving_a_cache.mdx

```diff
@@ -0,0 +1,107 @@
+
+- `Server.TLSCACertificateFile`
+  - The intermediate certificate from Let's Encrypt to establish the trust chain to a root certificate, usually named as `letsencrypt-intermediate.pem` or `chain.pem`
+  - Default location: `~/.config/pelican/certificates/tlsca.pem` for non-root users or `/etc/pelican/certificates/tlsca.pem` for root users.
+
+Since your TLS certificate is associated with your domain name, you will need to change the default hostname of Pelican server to be consistent. Set `Server.Hostname` to your domain name (e.g. `example.com`).
+
+## Serve a Cache
+
+> If you are running Pelican docker image to serve a cache, please refer to the [docker image documentation](./install/docker.mdx#run-pelican-cache-server).
+
+### Find a federation to join
+
+Before serving a cache, you need to find a Pelican federation to join in. If you are unfamiliar with the term **federation**, refer to [Useful Terminology](./client-usage.mdx#useful-terminology) before proceeding.
+
+If you don't have a federation in mind, the Open Science Data Federation (OSDF) is an example Pelican federation that you can join in for testing purposes. If you are interesting in serving an OSDF cache, refer to the [OSDF website](https://osg-htc.org/) for details.
+
+The federation discovery URL for OSDF is `osg-htc.org`. You may use this as your `<federation>` argument in the next section when launching your origin.
+
+### Launch the Cache
+
+To launch a pelican cache, run:
+
+```bash
+pelican cache serve -f <federation> 
+```
+
+Where:
+
+* `<federation>` is the URL to the federation the cache will be joining
+
+This will start a Pelican cache as a daemon process.
+
+### Additional arguments to launch a Cache
+
+This section documents the additional arguments you can pass to the command above to run the cache.
+
+* **-h or --help**: Output documentation on the `serve` command and its arguments.
+* **-p or --port**: Set the port at which the Pelican admin website should be accessible.
+
+* **--config**: Set the location of the configuration file.
+* **-d or --debug**: Enable the debugging mode, allowing for more verbose log
+* **-l or --log**: Set the location of the file where log messages should be redirected to and not output the the console.
+
+There are other configurations available to modify via the configuration file. Refer to the [Parameters page](./parameters.mdx) for details.
+
+## Test CacheFunctionality
+
+Once you have your cache set up, follow the steps below to test if your cache can access a file through a Pelican federation. 
+
+1. Have data available that is accessible via an origin. Since a cache only works with data available via an origin that’s in the same data federation, the following assumes that you have a an accessible namespace within the same data federation. For more information on how to set that up, refer to [Serving an Origin](./serving_an_origin.mdx). Ensure there is some public file available.
```

Comment:
```suggestion
1. Have data available and accessible via an origin. Since a cache only works with data available via an origin that’s in the same Pelican federation, the following assumes that you have a an accessible namespace within the same Pelican federation. For more information on how to set that up, refer to [Serving an Origin](./serving_an_origin.mdx). The following also assume that the object to test is public file. To set up a public file through a Pelican origin, refer to the [Configuration documentation](./parameters.mdx#Origin-Exports).
```